docs/develop/processing.md

```diff
@@ -59,16 +61,18 @@ import requests
 import tempfile
 import numpy as np
 
-def preprocess(self, model_input: Any) -> Any:
-    request = requests.get(model_input)
-    with tempfile.NamedTemporaryFile() as f:
-        f.write(request.content)
-        f.seek(0)
-        input_image = tf.image.decode_png(tf.io.read_file(f.name))
-    preprocessed_image = tf.keras.applications.resnet_v2.preprocess_input(
-        tf.image.resize([input_image], (224, 224))
-    )
-    return np.array(preprocessed_image)
+class Model:
```

Comment:
this also needs a predict to make sense I think. Or you need a `...` under `class Model:`

Reply:
Will add predict and load stubs so code doesn't show errors when copied like it would for ...